.github/workflows/build-and-test.yml

```diff
@@ -34,6 +34,35 @@ jobs:
     - name: Build all classes
       run: ./gradlew --stacktrace classes
 
+    - name: Install Node
+      uses: actions/setup-node@v4
+      with:
+        node-version: 20
+
+    - uses: pnpm/action-setup@v3
+      name: Install pnpm
+      with:
+        version: 8
+        run_install: false
```

Comment:
That's the [default behavior](https://github.com/pnpm/action-setup?tab=readme-ov-file#run_install) and should be dropped.